commons/src/main/java/org/infinispan/commons/jmx/JmxUtil.java

```diff
@@ -32,101 +33,99 @@
     * @return an instance of {@link javax.management.MBeanServer}
     */
    public static MBeanServer lookupMBeanServer(MBeanServerLookup lookup, Properties properties) {
+      Objects.requireNonNull(lookup);
```

Comment:
Technically not needed as the next line will throw a `NullPointerException` anyways.

Reply:
True. Meanwhile, that code is gone.